### PR TITLE
Bug 1152227 - [RTL][Notification] Support bi-direaction text in message detail, r= mikehenrty, gandalf

### PR DESCRIPTION
**Pull request by ian-liu**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=12e0cb32de070a23077e83d660227b549dbbb49b

File: apps/system/index.html [hunk 1]
```diff
@@ -1019,7 +1019,7 @@ <h1 data-l10n-id="data-roaming-enabled-title"></h1>
         <img id="toaster-icon" role="presentation" class="toaster-icon"/>
         <div id="notification-container">
           <div id="toaster-title" class="toaster-title" dir="auto"></div>
-          <div id="toaster-detail" class="toaster-detail"></div>
+          <div class="toaster-detail"><bdi id="toaster-detail"></bdi></div>
```
Comment (zbraniecki) [inline apps/system/index.html]: it seems that you should be able to just set dir="auto" on the <div>, which does the same thing and doesn't require additional markup (you will not need the <bdi> tag then)

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3561f0620fda7153dafde8420498cf15d07866ef

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fa39e94deb162bd41c6965236da97ec7b9e30298

File: apps/system/js/notifications.js [hunk 2]
```diff
@@ -468,9 +468,11 @@ var NotificationScreen = {
 
     notificationNode.appendChild(titleContainer);
 
+    var messageWrapper = document.createElement('bdi');
```
Comment (zbraniecki) [inline apps/system/js/notifications.js]: you don't need <bdi> element since you set dir="auto" on the toaster-detail it acts as a bdi.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9749c6c6b56c8aa779916b7300933a058ceacaed

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dbfed13a974da9710dec77134f9cde774f064743

**Pull request merged**